types/rimiti__stimmy/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for @rimiti/stimmy 1.8
+// Project: https://github.com/rimiti/stimmy
+// Definitions by: Dimitri DO BAIRRO <https://github.com/rimiti>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function replacer(start?: string, end?: string): (val: string, replace: {} | string[] | number[]) => string;
```

Comment:
This `replace` parameter is just equivalent to `{}` - what's the intent exactly?